CMakeLists.txt

```diff
@@ -108,7 +118,7 @@ lvk_set_folder(minilog "third-party")
 target_link_libraries(LVKLibrary PUBLIC minilog)
 target_include_directories(LVKLibrary PUBLIC "third-party/deps/src")
 
-if(WIN32 OR (UNIX AND NOT ANDROID))
+if(LVK_WITH_GLFW AND (WIN32 OR (UNIX AND NOT ANDROID)))
```

Comment:
Let's just do `if(LVK_WITH_GLFW)` here and disable GLFW in the code above for Android.